app/database/operator/wrapper/index.ts

```diff
@@ -1,38 +1,19 @@
 // Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
 // See LICENSE.txt for license information.
 
-import {Database} from '@nozbe/watermelondb';
-
-import {Operator} from '@database/operator/index';
+import Operator from '@database/operator/index';
```

Comment:
We won't be needing this wrapper, the DBManager will hold the reference to each DB Operator